CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
-<h1><span lang="en">Contributing</span> / <span lang="de">Mitarbeit</span></h1>
+# Contributing :tada:
+Thank you for wanting to help out! :heart:
 
-<span lang="en">Thank you for wanting to help out!</span> <em lang="de">Danke dass du mithelfen möchtest!</em>
+Danke dass du mithelfen möchtest!
+Die deutsche Version dieses Dokuments findest du [hier](./CONTRIBUTING_DE.md).
 
-<em lang="de">Die <a href="#deutsch">deutsche Übersetzung</a> findest du weiter unten.</em>
+# English :milky_way:
 
-## English
+## [Code of conduct](https://www.ecamp3.ch/en/code-of-conduct) :page_with_curl:
 
-### [Code of conduct](https://www.ecamp3.ch/en/code-of-conduct)
+## Workflow :gear:
+This is a basic overview of the workflow, i.e. how we work with the code of eCamp v3. More information about how to set up a development environment on your computer is in the [wiki](https://github.com/ecamp/ecamp3/wiki/installation).
+If something about the setup is unclear, or you run into an error, there are [discussions](https://github.com/ecamp/ecamp3/discussions) on GitHub for you to ask questions and ask for help. :computer:
+### Labels :label:
+Issues are marked with labels and some of them are not self-explanatory and are explained here:
+- **Type-Labels**:
+  Type labels follow the `type: *` format with the options `Frontend`, `Print`, `Deployment` & `API` the architecture for those are partially documented in the [wiki](https://github.com/ecamp/ecamp3/wiki/architecture-frontend)
+- **Needs prototype**: :bulb: If you have an idea how to solve this issue: we'd like to see it. This issue needs a prototype before actual implementation begins since the specifications are somewhat vague. A prototype can be many things, whether your prototype is a sketch, mockup, partial implementation or something else is up to you.
+- **Good first issue**: :green_heart: Beginner friendly issues.
+- **Feature request**: :rocket: An idea/request for a functionality, not ready to be implemented but ready to be discussed.
 
-### Git setup
+### :point_right: Starting with an issue 
+To get started, find an issue that interests you. If you are new, we recommend selecting a [Good first issue](https://github.com/ecamp/ecamp3/labels/Good%20first%20issue).
+Please note that for other issues we recommend ones with the label [Ready for Implementation](https://github.com/ecamp/ecamp3/issues?q=is%3Aopen+is%3Aissue+label%3A%22Ready+for+implementation%22), which signifies that the issue should have clear definitions. If you have any questions, feel free to ask.
```

Comment:
Does the Ready for Implementation label deserve to be described above? Or is it self-explanatory enough?

Reply:
I think it's self-explanatory and somewhat normal. If you see an issue with "Ready for implementation" it is clear what this means. Unlike (for example) "Needs prototype" where that could mean multiple things.